components/org.wso2.carbon.identity.context.rewrite.valve/src/main/java/org/wso2/carbon/identity/context/rewrite/valve/TenantContextRewriteValve.java

```diff
@@ -81,6 +84,10 @@ public void invoke(Request request, Response response) throws IOException, Servl
             }
         }
 
+        if (isTenantQualifiedUrlsEnabled) {
+            String tenantDomain = PrivilegedCarbonContext.getThreadLocalCarbonContext().getTenantDomain();
+            IdentityUtil.threadLocalProperties.get().put(TENANT_NAME_FROM_CONTEXT, tenantDomain);
```

Comment:
Looks like this TENANT_NAME_FROM_CONTEXT is read from somewhere else. As such
private static final String TENANT_NAME_FROM_CONTEXT = "TenantNameFromContext";
Needs to be either 
a) public so that any other place refer the same constant
b) use a common constant read from outside of this class

Reply:
Changed the implementation to read this constant from the public constant that is defined in the identity-core component.